portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,15 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Show/hide the description inside the element which has title as id.
  */
-function addRandomGreeting() {
-  const greetings = [
-    'I play tennis.',
-    '"An office is a place where dreams come true." — Michael Scott (The Office).',
-    '"A bug is never just a mistake. It represents something bigger. An error of thinking that makes you who you are." - Elliot (Mr Robot).',
-    'I have three dogs and one cat.'
-  ];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function toggleContent(title) {
+  var descr = document.getElementById(title).getElementsByClassName("description");
```

Comment:
Also there is not really any harm in writing out the variable name, so why not just call it `descriptionElement`?

Reply:
have you though what might happen here if you misspell the `title` argument and it doesn't find any element in the DOM?

Reply:
Yes, if the title is misspelled, it would give an error. Since I wrote the title argument when I call the function, I assumed that it is correct.
Would you suggest checking it anyway?

Reply:
I think the important thing here is to be aware. It depends a bit on programming standards and conventions what you want to do. E.g. usually we try to avoid any possibilities of a NullPointerException because they might stop your whole program in bad cases, miss cleanup steps and abort action in the middle of the logic. A safe way here would be to check it's truthy and only then do the rest of the logic. (Or more readable, return early out of the function if it is falsy/undefined/null)

Also shameless plug for TypeScript which would not have allowed you to write this ;)

Reply:
and then these become very nice test cases for unit tests.